metricbeat/module/system/network_summary/_meta/data.json

```diff
@@ -0,0 +1,245 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "system.network_summary",
+        "duration": 115000,
+        "module": "system"
+    },
+    "metricset": {
+        "name": "network_summary",
+        "period": 10000
+    },
+    "service": {
+        "type": "system"
+    },
+    "system": {
+        "network_summary": {
+            "icmp": {
+                "InAddrMaskReps": 0,
```

Comment:
is it common practice in metricbeat to use camel case for a set of 'undocumented' metrics? Do we want to convert to snake case (e.g. github.com/stoewer/go-strcase).

Reply:
So, my thinking was that since we're inserting "blind" metrics with no real opinionated manipulating of the data, keeping the key values the same as the their OS values might be a good idea. Let me hunt around the system module to see if we do this in other places.

Reply:
@exekias @jsoriano Do we have any precedence on using Pascal/CamelCase in events in metricbeat?

Reply:
We have done this when ingesting metrics in raw from a source, to avoid confusion. AWS cloudwatch is an example. That said, I don't have a strong opinion here